tests/test_operations.py

```diff
@@ -68,9 +80,9 @@ def test_cmd_start(state_dir, galaxy_yml, startup_config, free_port):
     assert result.exit_code == 0, result.output
     start_instance(state_dir, free_port)
     result = runner.invoke(galaxyctl, ['--state-dir', state_dir, 'status'])
-    for process in result.stdout.splitlines():
-        assert "RUNNING" in process or "STARTING" in process, process
-    assert (state_dir / "celery-beat-schedule").exists()
+    celery_beat_db_exists = wait_for_path(state_dir / "celery-beat-schedule", CELERY_BEAT_TIMEOUT)
```

Comment:
Since the `"celery-beat-schedule"` literal also occurs in `state.py`, maybe define it in `GalaxyCeleryBeatService` and reference it here?

Reply:
Fixed, plus I finally figured out why it was appending `.db` when I ran the tests locally and fixed that.